src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
So, afaik, `--no-cache-read` (ie the "global" form of the subsystem option) is applied recursively to all instances of the subsystem below that scope. Or, at least: I know that that is sufficient to disable caching everywhere.

The effect is almost identical to a `global_options` option:
```
# disables caching for all tasks
./pants --no-cache-read compile $t
# disables caching for tasks in compile
./pants compile  --no-cache-read $t
./pants --no-cache-compile-read compile $t
# disables caching for zinc
./pants compile.zinc  --no-cache-read $t
./pants --no-cache-compile-zinc-read compile $t
```